support-lib/jni/djinni_support.hpp

```diff
@@ -567,13 +567,45 @@ class JniEnum {
 
 protected:
     JniEnum(const std::string & name);
+    jclass enumClass() const { return m_clazz.get(); }
 
 private:
     const GlobalRef<jclass> m_clazz;
     const jmethodID m_staticmethValues;
     const jmethodID m_methOrdinal;
 };
 
+class JniFlags : public JniEnum {
```

Comment:
JniEnum as a member is not possible as the JniEnum constructor is protected.
I think implement m_staticmethValues in JniFlag could be a better choice.